vendor/assets/components/lightbox2/javascripts/lightbox.js

```diff
@@ -22,8 +22,8 @@
       this.alwaysShowNavOnTouchDevices = false;
       this.fadeDuration                = 20;
       this.fitImagesInViewport         = true;
-      // this.maxWidth                    = 800;
-      // this.maxHeight                   = 600;
+      this.maxWidth                    = window.innerWidth;
+      this.maxHeight                   = window.innerHeight;
```

Comment:
We should not be modifying the default values in the library itself.

Rather, wherever we instantiate the lightbox, we should provide the "window.innerWidth" and "window.innerHeight" options i.e. https://lokeshdhakar.com/projects/lightbox2/index.html#options